seatunnel-e2e/seatunnel-e2e-common/src/test/java/org/apache/seatunnel/e2e/common/AbstractContainer.java

```diff
@@ -42,15 +43,14 @@ public abstract class AbstractContainer {
     protected final String startModuleFullPath;
 
     public AbstractContainer() {
-        String[] modules = getStartModulePath().split(File.separator);
-        this.startModuleName = modules[modules.length - 1];
-        this.startModuleFullPath = PROJECT_ROOT_PATH + File.separator +
-            START_ROOT_MODULE_NAME + File.separator + getStartModulePath();
+        this.startModuleName = getStartModuleName();
+        this.startModuleFullPath = PROJECT_ROOT_PATH + File.separator + START_ROOT_MODULE_NAME + File.separator + getStartModuleName();
```

Comment:
change `getStartModuleName` to `this.startModuleName`

Reply:
done